site/src/pages/CreateWorkspacePage/CreateWorkspacePageView.tsx

```diff
@@ -301,7 +301,7 @@ export const CreateWorkspacePageView: FC<CreateWorkspacePageViewProps> = ({
 							</FormHelperText>
 						</div>
 
-						{permissions.createWorkspaceForUser && (
+						{permissions.createWorkspace && (
```

Comment:
this is now incorrect. We should only show this input if the user can create workspaces for another user